packages/plugin-nft-generation/src/actions/mintNFTAction.ts

```diff
@@ -0,0 +1,338 @@
+import {
+    Action,
+    composeContext,
+    Content,
+    elizaLogger,
+    generateObjectDeprecated,
+    HandlerCallback,
+    IAgentRuntime,
+    Memory,
+    ModelClass,
+    State,
+} from "@ai16z/eliza";
+import { createNFT } from "../handlers/createNFT.ts";
+import { verifyNFT } from "../handlers/verifyNFT.ts";
+import { sleep } from "../index.ts";
+import WalletSolana from "../provider/wallet/walletSolana.ts";
+import { PublicKey } from "@solana/web3.js";
+
+const mintTemplate = `Respond with a JSON markdown block containing only the extracted values. Use null for any values that cannot be determined.
```

Comment:
Thanks for doing this - would consider creating a template file and moving this over as is the norm for other plugins

Reply:
Alright, I'll get right on it.